### ISSUE TITLE
Does the "shuffle" really work?

### ISSUE THREAD

**Issue by bookong22** [speechbrain/speechbrain#1335]
yaml:
```
train_dataloader_opts:
    batch_size: !ref <batch_size>
    shuffle: True
    num_workers: !ref <num_workers>
```
fit:
 ```
abc_brain.fit(
        abc_brain.hparams.epoch_counter,
        train_data,
        valid_data,
        train_loader_kwargs=hparams["train_dataloader_opts"],
        valid_loader_kwargs=hparams["valid_dataloader_opts"],
    )
```

<img width="381" alt="9946a4f3ffcc48872f13b1fb59ebdec" src="https://user-images.githubusercontent.com/20372403/158926558-bac18efb-6a30-4954-b9de-755a17414fc2.png">

It seems that each epoch logged an error in the same place!

**Comment by bookong22** [comment]
I read the source code:
`if shuffle and not self.distributed_launch`
"shuffle" takes effect only in non-distributed mode!
sorry!
